scraper/pipelines.py

```diff
@@ -0,0 +1,26 @@
+from itemadapter import ItemAdapter
+
+
+class DatabasePipeline:
+    def __init__(self, url):
```

Comment:
Dit moet er uiteindelijk voor gaan zorgen dat alles wat uit de scraper komt in de database terecht komt